elasticdl/python/master/main.py

```diff
@@ -6,7 +6,11 @@
 from kubernetes.client import V1EnvVar
```

Comment:
Perhaps also change this for master in the same PR since the referenced issue is for fixing both of them?

Reply:
Do you mean the worker? The worker does not create argument collections such as `worker_args` or `container_args`.